plugins/performance-lab/includes/admin/load.php

```diff
@@ -230,6 +227,24 @@ function perflab_enqueue_features_page_scripts(): void {
 
 	// Enqueue the a11y script.
 	wp_enqueue_script( 'wp-a11y' );
```

Comment:
Do we need these enqueue now? as it already added through new script dependancy.

Reply:
We could remove it as it is not being used anywhere other than the AJAX script as it already added in its dependancy.